### PR TITLE
79 Add integration test for use of special chars in variable ID

### PR DESCRIPTION
**Pull request by sgnn7**
While we have a few unit tests around this, having a solid integration
test for odd characters in the ID will ensure that we do not get odd
behaviour when interacting with a rela Conjur server.

### What does this PR do?
- Makes Linux and Windows integration testing also use a variable with special characters in ID

### What ticket does this PR close?
Connected to #79 

#### Change log
- [ ] The CHANGELOG has been updated, or
- [x] This PR does not include user-facing changes and doesn't require a CHANGELOG update

#### Test coverage
- [x] This PR includes new unit and integration tests to go with the code changes, or
- [ ] The changes in this PR do not require tests

#### Documentation
- [ ] Docs (e.g. `README`s) were updated in this PR, and/or there is a follow-on issue to update docs, or
- [x] This PR does not require updating any documentation

### CODE REVIEW

File: examples/puppetmaster/policy.yml [hunk 1]
```diff
@@ -14,6 +15,11 @@
       resource: *db-password
       privilege: [ read, execute ]
 
+    - !permit
+      role: [ !layer ]
+      resource: *funky-variable
```
Comment (BradleyBoutcher) [inline examples/puppetmaster/policy.yml:20]: 🕺

**Review by BradleyBoutcher — APPROVED** [review]
LGTM

File: examples/puppetmaster/vagrant/5_get_puppet_artifacts.sh [hunk 2]
```diff
@@ -4,5 +4,8 @@
 
 set -eou pipefail
 
-echo "Getting C:\tmp\test.pem from Windows VM"
-vagrant powershell -c "cat \tmp\test.pem"
+echo 'Getting C:\tmp\creds1.pem from Windows VM'
```
Comment (doodlesbykumbi) [inline examples/puppetmaster/vagrant/5_get_puppet_artifacts.sh:7]: In this file `s/.pem/.txt`

**Review by doodlesbykumbi — COMMENTED** [review]
LGTM! Small mistake with `.pem` vs `.txt` in `examples/puppetmaster/vagrant/5_get_puppet_artifacts.sh`

**Review by doodlesbykumbi — APPROVED** [review]
LGTM!

**Pull request merged**